ui/appui-layout-react/src/test/state/internal/NineZoneStateHelpers.test.ts

```diff
@@ -14,3 +19,33 @@ describe("initSizeProps", () => {
     sut.should.eq(obj);
   });
 });
+
+describe("initRectangleProps", () => {
```

Comment:
Probably true, my intention was to avoid updating the original obj in case RectangleProps match. Added a test for that case (for which your simple impl would fail).